### DISCUSSION TITLE
Unable to install on M1 MacBook

### DISCUSSION THREAD

**Discussion by arujbansal** [royshil/obs-backgroundremoval#380] [General]
When I run,
`./.github/scripts/build-macos -c Release -t macos-x86_64
`

I get the error "Invalid value macos-x86_64 for option -t"

I didn't really understand how to install the plugin. Could someone please help me out?

**Comment by umireon** [comment]
Just download obs-backgroundremoval-1.1.0-macos-universal.pkg on https://github.com/royshil/obs-backgroundremoval/releases/tag/1.1.0 and install the pkg!

**Reply by arujbansal** [reply]
I tried using that installer first. There were no issues with the process but it doesn't show Background Removal as an option in my Effect Filters.

**Reply by arujbansal** [reply]
Update: Never mind, thanks! Updated from OBS 28.0.1 to the latest version and I can see it now.
I think you should add the installer to the project README. It makes the process a lot quicker and simpler for people new to installing OBS plugins.

**Reply by umireon** [reply]
It already exists but is too short that many people miss the link.
<img width="525" alt="スクリーンショット 2023-07-11 15 57 25" src="https://github.com/royshil/obs-backgroundremoval/assets/1067855/17806f86-814e-47f4-be28-96d5cc8570d9">

**Reply by arujbansal** [reply]
Oh you are right--I did miss that.

**Reply by umireon** [reply]
I will fix this part of README to appear to people prominently.
